pkg/services/ngalert/api/api_testing.go

```diff
@@ -77,3 +78,18 @@ func (srv TestingApiSrv) RouteTestRuleConfig(c *models.ReqContext, body apimodel
 		nil,
 	)
 }
+
+func (srv TestingApiSrv) RouteEvalQueries(c *models.ReqContext, cmd apimodels.EvalQueriesPayload) response.Response {
+	now := cmd.Now
+	if now.IsZero() {
+		now = timeNow()
+	}
+
+	evaluator := eval.Evaluator{Cfg: srv.Cfg}
+	evalResults, err := evaluator.QueriesAndExpressionsEval(c.SignedInUser.OrgId, cmd.Data, timeNow(), srv.DataService)
```

Comment:
I think maybe this should be `now` and not `timeNow()`

Reply:
(This fixes what I noted before based on my manual tests).

Reply:
thanks for finding it!